### ISSUE TITLE
Cannot create release (version 14.2.1)

### ISSUE THREAD

**Issue by marcuspoehls** [release-it/release-it#716]
Hey Lars,

I’m running into issues with failed commits when tagging a new release using `releasse-it v14.2.1`.

My issues may relate to my local system. I can’t find the source of this problem in release-it. That’s the reason I’m looking for fresh eyes hinting me at a misconfiguration on my machine. Maybe it’s an issue with release-it (but I don’t think it is).

Here’s a snippet from the release-it CLI. I’ve stripped the commits to the last three. When selecting version `3.13.0` I’m receiving the "empty changeset" message which may be correct. The commit containing the new release version `package.json` works fine. Then, I’m seeing an error related to the git commit command. Sorry for the german part inside the snippet. Looks like release-it uses a different git configuration that my default "en" locale 😄 

```bash
* add software-vendors route (f0d0934e)
* SqlServerExpressCpuLimit (a5c6e132)
* formatting (54287105)

? Select increment (next version): minor (3.13.0)

Empty changeset

? Commit (Release 3.13.0)? Yes

Auf Branch develop
Ihr Branch ist 90 Commits vor 'origin/develop'.
  (benutzen Sie "git push", um lokale Commits zu publizieren)

nichts zu committen, Arbeitsverzeichnis unverändert
ERROR Error: Command failed with exit code 1: git commit --message Release 3.13.0
Auf Branch develop
Ihr Branch ist 90 Commits vor 'origin/develop'.
  (benutzen Sie "git push", um lokale Commits zu publizieren)

nichts zu committen, Arbeitsverzeichnis unverändert
Rolling back changes...
➜  il-backend git:(develop)
```

So i tried running the `git commit --message Release 3.13.0` separately in my terminal. This fails as well. Are the quotes required and missing?

```bash
this is my 
➜  il-backend git:(develop) git commit --message Release 3.13.0
error: pathspec '3.13.0' did not match any file(s) known to git
```

Do you have an idea why this happens?

**Comment by webpro** [comment]
```
ERROR Error: Command failed with exit code 1: git commit --message Release 3.13.0
```

```
➜  il-backend git:(develop) git commit --message Release 3.13.0
```

The `--message` argument definitely should have quotes around "Release 3.13.0". And the question is indeed why they are missing... It's the default commit message, so I'm a bit surprised as to how this can happen. What OS and what version of Git are you using?

For background information, internally, release-it sends this to [execa](https://github.com/sindresorhus/execa#execafile-arguments-options) which should handle the quoting (and afaic does this very well). I remember issues (before using `execa`) with quotes and multi-line changelogs, so issues around tag annotations and commit messages have been tested quite extensively in the past. Any related changes go back 8 months.

**Comment by webpro** [comment]
```
nichts zu committen, Arbeitsverzeichnis unverändert
ERROR Error: Command failed with exit code 1: git commit --message Release 3.13.0
```

This command could fail, which might not be an issue, if there are no changed files. E.g. the `package.json` already contains the new version number (hence the "Empty changeset" message). So I think the issue is probably related to the fact that there is nothing to commit.

And the error log probably does not contain quotes around the git commit message as release-it does not have them when logging commands, so I think this led us thinking in the wrong direction.

**Comment by marcuspoehls** [comment]
@webpro I’m using

- git version `2.29.2: installed and updated via Homebrew
- OS: macOS Catalina 10.15.7

I noticed while pushing the git changes to the server that the tags are created correctly. The commit for the version change in `package.json` is also successful. I can revert the change using VS Code "undo last commit".

What I’m thinking about: looks like the commit is done twice. I didn’t start a second release-it instance in another terminal tab 😄

**Comment by webpro** [comment]
Well, I'm inclined to think release-it does not perform two commits, never heard of this issue before.

Maybe there is some git hook or npm script (e.g. `npm run postversion`) installed that causes this?

**Comment by marcuspoehls** [comment]
@webpro There are no git or NPM hooks in the project. I’ll investigate the behavior a bit more during the week. I suspect to be something odd with git on my machine. Because it switches languages when displaying the error. I’m sharing my findings here.

Thank you for your thoughts and ideas Lars!

**Comment by marcuspoehls** [comment]
I fixed it. Well, it just worked correctly again 😄 

Ok, here’s what I think went wrong on my machine: I ran `brew update` on my system and this probably upgraded git and other tools. The open terminal tab for my project didn’t reload the .zshrc file to pick up any changes or reference the new tool paths. However, I restarted my machine and created a new release using `release-it`. It works, without issues.

Sorry for the fuzz and thank you for your help, Lars. I really appreciate it especially because it wasn’t reproducible for you. Thanks, buddy!
